relay-dynamic-config/src/project.rs

```diff
@@ -90,6 +91,13 @@ impl ProjectConfig {
         metrics::convert_conditional_tagging(self);
         defaults::add_span_metrics(self);
     }
+
+    /// Retrieves the measurements from project config and falls back on global config if missing.
+    pub fn measurements<'a>(&'a self, global_config: &'a Arc<GlobalConfig>) -> &MeasurementsConfig {
```

Comment:
having a reference to an Arc feels pretty ugly, but I did it to keep consistent since the field value is an Option<MeasurementsConfig>

should I perhaps instead make the field value an option<arc<x>> ? or is there a better way altogether